### DISCUSSION TITLE
Navbar not coming properly in mobile phone view

### DISCUSSION THREAD

**Discussion by Karan-S-dev** [facebook/docusaurus#8181] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/88817564/194254635-0ce41977-28f0-4e4c-91ca-c8b852c47e4d.png)

If I go to mobile phone response, the navbar is coming inside those three buttons which is present on the top left side. This is the functionality I am expecting. But in my docusaurus project it is not coming.

![image](https://user-images.githubusercontent.com/88817564/194256086-9bd31651-b20d-4aa6-bd9b-be689c82f812.png)

I have made many changes in the docusaurus.config.js file so I am not able to track back what I changed. Is there any config i have to add in docusaurus.config.js file to achieve that feature?

**Comment by slorber** [comment, marked as answer]
🤷‍♂️ if the hamburger is not displayed, it's likely due to your site customizations. You didn't share your site URL so it's impossible to help you

**Reply by Karan-S-dev** [reply]
- Any other files needed , i can share.

**Reply by slorber** [reply]
you have added custom CSS that makes this button hidden

![CleanShot 2022-10-06 at 11 19 36@2x](https://user-images.githubusercontent.com/749374/194275915-ea52e8aa-db69-4154-9921-a3fb6ed692c1.png)
